pkg/operator/manualapprovalgate.go

```diff
@@ -44,3 +57,99 @@ func EnsureManualApprovalGateExists(clients mag.ManualApprovalGateInterface, nam
 	})
 	return ks, err
 }
+
+func StartApprovalGatePipeline() {
+	cmd.MustSucceed("tkn", "pipeline", "start", "manual-approval-pipeline")
+	log.Println("Waiting 10sec to start the pipeline")
+	cmd.MustSuccedIncreasedTimeout(time.Second*130, "sleep", "10")
+}
+
+func GetApprovalTaskList() []TaskInfo {
+	output := cmd.MustSucceed("opc", "approvaltask", "list").Stdout()
+	tasklist := strings.Split(output, "\n")
+	var tasks []TaskInfo
```

Comment:
IMHO it would be better to use golang API here than to parse output.

Reply:
Done